CHANGELOG.md

```diff
@@ -1,3 +1,64 @@
+## v0.33.0 (2024-09-25)
+
+#### Notes:
+
+* The `--pubsub-topic` CLI configuration has been deprecated and support for it will be removed on release v0.35.0. In order to migrate, please use the `--shard` configuration instead. For example, instead of `--pubsub-topic=/waku/2/rs/<CLUSTER_ID>/<SHARD_ID>`, use `--cluster-id=<CLUSTER_ID>` and `--shard=<SHARD_ID>`
+* The `--rest-private` CLI configuration is no longer supported. Please remove any reference to it when running your nodes
```

Comment:
`no longer supported` Does that mean deprecated or removed?

Reply:
It means removed! Will change the wording